blockchain/README.md

```diff
@@ -18,66 +18,6 @@ If you have set your password token you can simply start the cluster `$ ./start-
 
```

Comment:
The lines above also have to be removed, we don't use a private docker registry anymore (images are on dockerhub). The `./start-service.sh`script can be removed, since it uses a docker-compose that is not located here anymore. 

Rather link the documentation here https://github.com/openkfw/TruBudget/tree/main/docker-compose and mention that this way you can start just the blockchain with `cp .env_example .env && docker-compose --project-directory . -f blockchain/docker-compose.alphanode.yml up`